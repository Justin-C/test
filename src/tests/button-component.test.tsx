import React from 'react';

import renderer from 'react-test-renderer';
import Button from '../components/button';

describe('button component test', () => {
  it('should render button', () => {
    const tree = renderer
      .create(<Button handleClick={() => {}} buttonText="test" isEnabled={true} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
