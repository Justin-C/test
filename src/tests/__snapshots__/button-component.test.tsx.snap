// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button component test should render button 1`] = `
<button
  className="undefined button"
  disabled={false}
  onClick={[Function]}
>
  test
</button>
`;
